### ISSUE TITLE
Use gopls@latest rather than master

### ISSUE THREAD

**Issue by banks** [microsoft/vscode-go#2671]
Based on discussion here: https://github.com/golang/go/issues/33221#issuecomment-514256469 the `gopls` authors suggest that vscode should be installing `latest` rather than `master` as they are more confident about the stability of `latest` for general use.

I don't personally have a horse in the race - it happened that I was hitting an issue that needed a fix from master anyway but I said I'd open this dialogue so the teams could decide between them what was the best policy here.

As far as I can see the gopls path here  https://github.com/microsoft/vscode-go/blob/81bc2fa8fc5940e70059201ddfc00af8e7f20e4f/src/goInstallTools.ts#L43 is just passed to `go get -u` which would pull master.

CC @stamblerre who suggested this request.

**Comment by ramya-rao-a** [comment]
Thanks for logging this issue @banks

@stamblerre In some issues we ask users to get the latest of gopls to get bug fixes that were recently made. Would you be ensuring that `@latest` would point to the right commit? Would there be a case where we want users to use master?

Because, from our side the suggestion would be to use `Go: Install/Update Tools` command to update the `gopls`. If this points to latest, and in one of GitHub issues you ask user to use the master, then there might be some confusion

**Comment by andrewrynhard** [comment]
FWIW VScode for Golang was nearly unusable until I used `latest`.

**Comment by oneslash** [comment]
I would also be for using `@latest`, since it is stable

**Comment by stamblerre** [comment]
Yes, latest should be the recommended because we will be able to guarantee that it is more stable than master (which can fluctuate, and may include breaking changes). I have not been asking people to upgrade to master lately, since it is typically less stable. If we do recommend it, we can ask people to update on the command line.

I can work on making this change.

**Comment by ramya-rao-a** [comment]
@stamblerre Would #2700 fix the current issue here?

**Comment by ramya-rao-a** [comment]
This should now be fixed in the latest version (0.11.5) of the extension
